### PR TITLE
Added new rules for CMP banners

### PR DESCRIPTION
**Pull request by pritam2406**
Added rules for 3 websites.

### CODE REVIEW

File: src/util/cmp_banners.ts [hunk 1]
```diff
@@ -128,4 +128,58 @@ export const banners = [
       },
     }
   },
+
+  // https://trustarc.com/
```
Comment (antoniojtorres) [inline src/util/cmp_banners.ts]: @pritam2406 Is this the website you used to develop and verify the rule?

**Review by antoniojtorres — CHANGES_REQUESTED** [review]
Would like to test this on at least 3 domains to ensure it's working. See the document that @rag24 is working on.

File: src/util/cmp_banners.ts (hunk 1 above)
Comment (pritam2406) [reply src/util/cmp_banners.ts]: yes.

**Review by antoniojtorres — COMMENTED** [review]
@pritam2406 Great progress. I tested all three examples and was able to reproduce. Flickr seems straight forward, we should check if a distinct element exists using locator().count() to determine which variant we're in. I think the onetrust one has that already  you can reference.

**Comment by antoniojtorres** [comment]
@pritam2406 Thank you! This PR is getting pretty big. Let's stop where it's at. I have a refactor coming that I'll need help with, it'll allow us to submit handle CMPs with many different variants, and it makes them individually testable. I'll merge this as is and then make a new branch for that. Thanks again

**Review by antoniojtorres — APPROVED** [review]
Approving without testing to make room for incoming refactor that adds testing.

**Pull request merged**